raiden_contracts/tests/test_channel_settle_unlock_state.py

```diff
@@ -37,8 +36,6 @@ def test_settlement_outcome(
     def f(
             participants,
             channel_values,
-            expected_settlement0,
-            expected_settlement_onchain0,
```

Comment:
This removal indicates that this fixture was not checking enough.  The issue to fix this https://github.com/raiden-network/raiden-contracts/issues/866 .